tools/ci/sizeCheck.cjs

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs')
+const path = require('path')
+
+const THRESHOLD = 1.05
+const workspacePath = path.dirname(__filename)
+const weblibPath = path.join(workspacePath, '../..', 'build/cmake-wasm/weblib')
+const sizeFilePath = path.join(weblibPath, 'size.txt')
+const sizeDownloadedFilePath = path.join(weblibPath, 'size_downloaded.txt')
+
+const sizeData = fs.readFileSync(sizeFilePath, 'utf8')
+const sizeLines = sizeData.split('\n').map((line) => line.trim())
+
+const downloadedData = fs.readFileSync(sizeDownloadedFilePath, 'utf8')
+const downloadedLines = downloadedData.split('\n').map((line) => line.trim())
+
+sizeLines.forEach((sizeLine) => {
+  if (!sizeLine) return
+  const [file, actualSize] = sizeLine.split(' ')
+
+  const downloadedLine = downloadedLines.find((dlLine) => {
+    const regex = new RegExp(`^${file}\\s+\\d+`)
+    return regex.test(dlLine)
+  })
```

Comment:
Ez nem egyszerűbb így?
```js
const regex = new RegExp(`^${file}\\s+\\d+`)
const downloadedLine = downloadedLines.find(regex.test)
```

Reply:
const downloadedLine = downloadedLines.find(regex.test)
TypeError: Method RegExp.prototype.test called on incompatible receiver undefined

However, I don't think this tool is worth any further extra effort.